packages/commonwealth/client/scripts/controllers/server/threads.ts

```diff
@@ -699,10 +699,11 @@ class ThreadsController {
   }
 
   public async fetchThreadsFromId(
-    ids: Array<number | string>
+    ids: Array<number | string>,
+    chain?: string
   ): Promise<Thread[]> {
     const params = {
-      chain: app.activeChainId(),
+      chain: chain ? chain : app.activeChainId(),
```

Comment:
Directly pass in chain if given (because now, we may be on the root URL via FeedPage

Reply:
This is fine and confirms to the latest code.